examples/ClusterHelloWorld/Node2/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -----------------------------------------------------------------------
+//   <copyright file="Program.cs" company="Asynkron HB">
+//       Copyright (C) 2015-2017 Asynkron HB All rights reserved
+//   </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using Messages;
+using Proto;
+using Proto.Cluster;
+using Proto.Remote;
+using ProtosReflection = Messages.ProtosReflection;
+
+namespace Node2
+{
+   
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Serialization.RegisterFileDescriptor(ProtosReflection.Descriptor);
+            var props = Actor.FromFunc(ctx =>
+            {
+                switch (ctx.Message)
+                {
+                    case HelloRequest msg:
+                        ctx.Respond(new HelloResponse
+                        {
+                            Message = "Hello from node 2"
+                        });
+                        break;
+                    default:
+                        break;
+                }
+                return Actor.Done;
+            });
+
+            Remote.RegisterKnownKind("HelloKind", props);
+            Remote.Start("127.0.0.1", 12000);
+            Cluster.Start("MyCluster",null);

```

Comment:
Boot up the cluster. we don't have a consul provider impl yet. thus null (which would fail)